scripts/version-update.js

```diff
@@ -45,4 +45,4 @@ export const VERSION = '${pjson.version}';
 
 const fileUrl = path.join(appRoot, "src", "version.ts")
 
-fs.writeFileSync(fileUrl, content.replace(/\n/g, os.EOL));
+fs.writeFileSync(fileUrl, content.replace(/\n/g, "\n"));
```

Comment:
```suggestion
fs.writeFileSync(fileUrl, content));
```
no need to replace `\n` by `\n`.